proof/append.go

```diff
@@ -160,6 +160,47 @@ func CreateTransitionProof(prevOut wire.OutPoint,
 		TapSiblingPreimage: params.TapscriptSibling,
 	}
 
+	if proof.Asset.IsTransferRoot() {
+		stxoInclusionProofs := make(
+			map[asset.SerializedKey]commitment.Proof,
+			len(proof.Asset.PrevWitnesses),
+		)
+		for _, wit := range proof.Asset.PrevWitnesses {
+			if wit.PrevID == nil {
+				return nil, fmt.Errorf("witness %v has no "+
+					"prevID", wit)
+			}
+
+			spentAsset := &asset.Asset{
```

Comment:
Shouldn't this be using the constructor to make alt leaves?

Reply:
:100: I think that was already the case when you reviewed this, as it says this code was already outdated even before I made revisions... weird.
Anyway, yes. :+1:

Reply:
This landed in the last commit, should probably be moved to this one.